spatial-tracking-explainer.md

```diff
@@ -343,82 +343,80 @@ interface XRInputPose {
 };
 
 //
-// Coordinate System
+// Space
 //
 
-[SecureContext, Exposed=Window] interface XRCoordinateSystem {
-  Float32Array? getTransformTo(XRCoordinateSystem other);
+[SecureContext, Exposed=Window] interface XRSpace : EventTarget {
+  Float32Array? getTransformTo(XRSpace other);
 };
 
 //
-// Frame of Reference
+// Reference Space
 //
 
-enum XRFrameOfReferenceType {
+enum XRReferenceSpaceType {
   "stationary",
   "bounded",
   "unbounded"
 };
 
-dictionary XRFrameOfReferenceOptions {
-  required XRFrameOfReferenceType type;
+dictionary XRReferenceSpaceOptions {
+  required XRReferenceSpaceType type;
 };
 
-[SecureContext, Exposed=Window] interface XRFrameOfReference : EventTarget {
-  readonly attribute XRCoordinateSystem coordinateSystem;
-  
+[SecureContext, Exposed=Window] interface XRReferenceSpace : XRSpace {  
   attribute EventHandler onreset;
 };
 
 //
-// Stationary Frame of Reference
+// Stationary Reference Space
 //
 
-enum XRStationaryFrameOfReferenceSubtype {
+enum XRStationaryReferenceSpaceSubtype {
   "eye-level",
   "floor-level",
   "position-disabled"
 }
 
-dictionary XRStationaryFrameOfReferenceOptions : XRFrameOfReferenceOptions {
-  required XRStationaryFrameOfReferenceSubtype subtype;
+dictionary XRStationaryReferenceSpaceOptions : XRReferenceSpaceOptions {
+  required XRStationaryReferenceSpaceSubtype subtype;
 };
 
 [SecureContext, Exposed=Window]
-interface XRStationaryFrameOfReference : XRFrameOfReference {
-  readonly attribute XRStationaryFrameOfReferenceSubtype subtype;
+interface XRStationaryReferenceSpace : XRReferenceSpace {
+  readonly attribute XRStationaryReferenceSpaceSubtype subtype;
 };
 
 //
-// Bounded Frame of Reference
+// Bounded Reference Space
 //
 
 [SecureContext, Exposed=Window]
-interface XRBoundedFrameOfReference : XRFrameOfReference {
+interface XRBoundedReferenceSpace : XRReferenceSpace {
   readonly attribute FrozenArray<DOMPointReadOnly> boundsGeometry;
 };
 
 //
-// Unbounded Frame of Reference
+// Unbounded Reference Space
 //
 
 [SecureContext, Exposed=Window] 
-interface XRUnboundedFrameOfReference : XRFrameOfReference {
+interface XRUnboundedReferenceSpace : XRReferenceSpace {
 };
 
 //
 // Events
 //
 
 [SecureContext, Exposed=Window,
- Constructor(DOMString type, XRFrameOfReferenceEventInit eventInitDict)]
-interface XRFrameOfReferenceEvent : Event {
-  readonly attribute XRFrameOfReference frameOfReference;
+ Constructor(DOMString type, XRReferenceSpaceEventInit eventInitDict)]
```

Comment:
Does this need to be an `XRSpaceEventInit` or is this ok as is?

Reply:
This is OK as-is because even though the `XRSpace` is the thing that directly extends the `EventTarget` the events themselves can only ever fire from the `XRReferenceSpace`